studio/components/interfaces/BillingV2/Subscription/CostControl/CostControl.tsx

```diff
@@ -0,0 +1,115 @@
+import { useParams } from 'common'
+import ShimmeringLoader from 'components/ui/ShimmeringLoader'
+import { useProjectSubscriptionV2Query } from 'data/subscriptions/project-subscription-v2-query'
+import { BASE_PATH, PRICING_TIER_PRODUCT_IDS } from 'lib/constants'
+import Image from 'next/image'
+import { useSubscriptionPageStateSnapshot } from 'state/subscription-page'
+import { Alert, Button } from 'ui'
+import SpendCapSidePanel from './SpendCapSidePanel'
+
+export interface CostControlProps {}
+
+const PLAN_NAME: any = {
+  tier_team: 'Team',
+  tier_enterprise: 'Enterprise',
+}
+
+const CostControl = ({}: CostControlProps) => {
+  const { ref: projectRef } = useParams()
+
+  const snap = useSubscriptionPageStateSnapshot()
+  const { data: subscription, isLoading } = useProjectSubscriptionV2Query({ projectRef })
+
+  const currentTier = subscription?.tier?.supabase_prod_id ?? ''
+  const isUsageBillingEnabled = subscription?.usage_billing_enabled ?? false
+
+  const canChangeTier = ![
+    PRICING_TIER_PRODUCT_IDS.TEAM,
+    PRICING_TIER_PRODUCT_IDS.ENTERPRISE,
+  ].includes(currentTier)
+
+  return (
+    <>
+      <div className="grid grid-cols-12 gap-6">
+        <div className="col-span-12 lg:col-span-5">
+          <div className="sticky top-16">
+            <p className="text-base">Cost control</p>
+            <p className="text-sm text-scale-1000">Some description text here</p>
```

Comment:
What about moving the text 

> You can control whether your project is charged for additional usage beyond the
> included quota of your subscription plan. If you need to go beyond the included
> quota, simply switch off your spend cap to pay for additional usage.

here? Saves us vertical space and we have enough space on that side